tensorflow/compiler/tf2tensorrt/convert/convert_nodes.h

```diff
@@ -182,10 +182,20 @@ class TRT_ShapedWeights {
 
   const Tensor& GetTensor() const { return tensor_; }
 
-  // Returns the raw pointer to the underlying buffer which holds the weights
-  // value.
-  void* GetValues() const {
-    return const_cast<char*>(tensor_.tensor_data().data());
+  // Returns a pointer of type T to the underlying buffer of the tensor.
+  template <typename T>
+  const T* GetValues() const {
```

Comment:
So it actually returns a pointer not values. Would it be better to rename this to GetPointer?